internal/filter/filter.go

```diff
@@ -150,48 +148,49 @@ func ListModulesForNMC(_ context.Context, obj client.Object) []reconcile.Request
 }
 
 func skipNodeHeartbeat() predicate.Predicate {
```

Comment:
We need to rename this function to something that reflects more of what it is doing.